src/dataflow/src/render/upsert.rs

```diff
@@ -185,127 +184,29 @@ where
 /// whereas the input stream is `Stream<G, (key, Option<value>, Timestamp)>
 fn apply_linear_operators<G>(
     stream: &Stream<G, (Row, Option<Row>, Timestamp)>,
-    linear_operator: &mut Option<LinearOperator>,
-    src_type: &RelationType,
+    _linear_operator: &mut Option<LinearOperator>,
```

Comment:
we might as well just get rid of these args?

Reply:
It's not that important to remove them here since I'll probably be doing it when merging arrangement-less upsert anyway.

Reply:
I had thought to leave them in to communicate that an updated implementation would take them as arguments, but I guess the method goes away too pretty soon. I'll probably want to rewrite it for MFP stuff quite soon.

Reply:
Like, this PR could strip them all out, up a few function level (`pre_arrange_from_upsert_transforms` doesn't need them either), and then the next PR could add them all back in. Idk.